include/STREAM/Settings/OptionConstants.enum.hpp

```diff
@@ -13,3 +13,29 @@ enum eqterm_E_field_eqn {
     UQTY_E_FIELD_EQN_CIRCUIT=101
 };
 
+
+enum Conf_Time_type
+{
+	CONF_TIME_BOHM      = 1,
+	CONF_TIME_INTOR     = 2,
+	CONF_TIME_ITER89    = 3,
+	CONF_TIME_ITER97    = 4,
+	CONF_TIME_IPB98     = 5,
+	CONF_TIME_ITER89_OL = 6,
+	CONF_TIME_EIV1      = 7,
+	CONF_TIME_EIV2      = 8,
+	CONF_TIME_GOLDSTONE = 9,
+	CONF_TIME_KAYE_BIG  = 10,
+	CONF_TIME_CY        = 11,
+	CONF_TIME_OS_OL     = 12,
+	CONF_TIME_RL_OL     = 13
+};
+
+
+enum Conf_Time_smoothless // Work in progress
+{
+	CONF_TIME_INVERSE_SUM = 1,
+	CONF_TIME_EXP_SUM = 2,
+	CONF_TIME_TANH_SUM = 3,
+	CONF_TIME_SUM = 4
+};
```

Comment:
is this work in progress? ready for MR? if not ready, it should be removed

Reply:
This has been removed